content/en/blogs/install-nfs-server-client-for-kubesphere-cluster.md

```diff
@@ -0,0 +1,347 @@
+
+   ```
+   NAME                                                 READY   STATUS    RESTARTS   AGE
+   calico-kube-controllers-59d85c5c84-ckrpb             1/1     Running   0          16m
+   calico-node-td5cw                                    1/1     Running   0          16m
+   calico-node-v2w2x                                    1/1     Running   0          16m
+   calico-node-xwrd5                                    1/1     Running   0          16m
+   coredns-74d59cc5c6-b764l                             1/1     Running   0          16m
+   coredns-74d59cc5c6-p47m7                             1/1     Running   0          16m
+   kube-apiserver-client1                               1/1     Running   0          16m
+   kube-controller-manager-client1                      1/1     Running   0          16m
+   kube-proxy-2t7n9                                     1/1     Running   0          16m
+   kube-proxy-4wkcr                                     1/1     Running   0          16m
+   kube-proxy-f5f67                                     1/1     Running   0          16m
+   kube-scheduler-client1                               1/1     Running   0          16m
+   metrics-server-5ddd98b7f9-lndw6                      1/1     Running   0          12m
+   nfs-client-nfs-client-provisioner-6fc95f4f79-92lsh   1/1     Running   0          16m
+   nodelocaldns-4gfhf                                   1/1     Running   0          16m
+   nodelocaldns-f55st                                   1/1     Running   0          16m
+   nodelocaldns-swprm                                   1/1     Running   0          16m
+   openebs-localpv-provisioner-84956ddb89-q5x6c         1/1     Running   0          16m
+   openebs-ndm-m2g44                                    1/1     Running   0          16m
+   openebs-ndm-mb49d                                    1/1     Running   0          16m
+   openebs-ndm-operator-6896cbf7b8-4h8jz                1/1     Running   1          16m
+   snapshot-controller-0                                1/1     Running   0          11m
+   ```
+
+### KubeSphere console
+
+1. The `ks-console` Service is being exposed via NodePort. Log in to the console at `<node IP>:30880` with the default account and password (`admin/P@88w0rd`). You may need to open the port in your security groups and configure relevant port forwarding rules depending on your environment.
+
+2. Click **Platform** in the top left corner and go to **Cluster Management**. In **Storage Classes** under **Storage**, you can see two storage classes:
+
+   ![nfs-storage-class](/images/blogs/en/install-nfs-server-client-for-kubesphere-cluster/nfs-storage-class.png)
+
+3. Go to **Pods** under **Application Workloads**, the Pod of `nfs-client` is also functioning well in the `kube-system` namespace.
+
+   ![nfs-client-pod](/images/blogs/en/install-nfs-server-client-for-kubesphere-cluster/nfs-client-pod.png)
```

Comment:
should we creating a PVC or Pod to prove the NFS sc is okay?

Reply:
Sure, but this will not be covered in detail. The blog mainly talks about the installation part instead of how to create PVCs on the console. Otherwise, the blog will be too long.